components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/userstore-config-finish.jsp

```diff
@@ -71,7 +72,7 @@
                         }
                     }
                     propertyDTO.setName(CharacterEncoder.getSafeText(request.getParameter("propertyName_" + i)));
-                    propertyDTO.setValue(value);
+                    propertyDTO.setValue(StringEscapeUtils.unescapeXml(StringEscapeUtils.unescapeXml(value)));
```

Comment:
Better to see why the values given through the UI are getting escaped multiple times rather than unescape from here.